modules/list_books.js

```diff
@@ -0,0 +1,72 @@
+import { DateTime } from './luxon.js';
+
+export default function dynamicLoad(awesome) {
+  const body = document.querySelector('body');
+  if (document.querySelector('.list')) {
+    body.removeChild(document.querySelector('.list'));
+  }
```

Comment:
- [ ] Please, It's one of the project requirements you use `arrow` syntax for functions instead of JavaScript Function Expressions because in this project you learn how to use ES6 new features. Kindly make this :wink: 💯 

- For more documentation, you can visit this [link](https://www.w3docs.com/learn-javascript/function-expressions.html)